src/test/polymer/expression-scanner_test.ts

```diff
@@ -54,23 +47,25 @@ suite('ExpressionScanner', () => {
       `;
       const underliner = CodeUnderliner.withMapping('test.html', contents);
       const document = new HtmlParser().parse(contents, 'test.html');
-      const visit = async(visitor: HtmlVisitor) => document.visit([visitor]);
 
-      const expressions = await scanner.scan(document, visit);
+      const results = await scanForExpressions(document);
+      const expressions = results.expressions;
+
+      assert.deepEqual(results.warnings, []);
       assert.deepEqual(
           await underliner.underline(expressions.map((e) => e.sourceRange)), [
             `
             <div id="{{foo}}"></div>
-                    ~~~~~~~~~`,
+                       ~~~`,
```

Comment:
I'm not really sure we want to exclude the braces from the source range. What if we want to warn on a two-way binding to a non-notifying property? We'd want to underline the curly-braces.